### PR TITLE
Add shorter command

### PR DESCRIPTION
**Pull request by benwtks**
Added a shorter command so that you can run `:S` instead of `:Spotify`

### CODE REVIEW

File: plugin/spotify.vim [hunk 1]
```diff
@@ -305,4 +305,5 @@ call s:SpotifyClearCache()
 let s:SpotifyOpenTrackUri = function( s:GetOS() . "UriOpener")
 
 command! -nargs=* Spotify call SpotifyTrackSearch("<args>")
+command! -nargs=* S call SpotifyTrackSearch("<args>")
```
Comment (watkins) [inline plugin/spotify.vim]: If you're not sure about using S as the shorter command, please suggest what should be used instead and I'll push a commit changing the command.

**Comment by takac** [comment]
This is unnecessary, you should be using tab completion to act as a shortcut to this command.

**Comment by watkins** [comment]
Good point, although when I put `:S` I have to tab twice before I get `:Spotify` as the first tab gives me `:Sexplore`. This means that using tab takes 3 key presses (S, tab, tab). In comparison, :S only takes 1.

It's a minute difference but one that I expect a lot of vim uses, who regularly try to shorten things, would appreciate. [Have a look at this for instance](https://youtu.be/SkdrYWhh-8s).

**Comment by takac** [comment]
Using `:S` as a shortcut is not only going to be confusing for users of multiple plugins but frankly harmful to the vim plugin ecosystem. If every plugin tried to sit on the shortest one letter command there would be constant collisions and issues.

If you want a shorter way to access the search make a binding in your `.vimrc`, either with a command or even better set up a `nnoremap`.

```
nnoremap <leader>s :Spotify 
command! -nargs=* S call SpotifyTrackSearch("<args>")
```

Also you should only have to type `:Sp yeezus`, vim will guess the rest. 1 more keystroke than `:S` but far more intuitive and less destructive to the plugin ecosystem.

**Comment by benwtks** [comment]
Good point, I hadn't considered that

**Pull request closed without merging**